client/my-sites/themes/theme-activation-confirmation-modal.scss

```diff
@@ -0,0 +1,40 @@
+.theme-activation-confirmation-modal {
+	min-height: 90px;
+	max-width: 400px;
+	margin: 0 auto;
+
+	@include breakpoint-deprecated( '<480px' ) {
+		box-sizing: border-box;
+		max-width: 100%;
+
+		+ .dialog__action-buttons .button {
+			display: block;
+			margin: 0 auto 10px;
+			width: 100%;
+		}
+	}
+
+	.form-label {
+		font-weight: 400;
+	}
+
+	.theme-activation-confirmation-modal__homepage-hint {
+		margin-left: 20px;
+		font-size: 0.875rem;
```

Comment:
Can this use `font-size: $font-body-small` instead?

Reply:
I don't think these styles are even used - they're from the old stylesheet, but there's no corresponding element with a class name of `*-hint`, so I've just removed them.